Resources/Prototypes/Loadouts/Jobs/Medical/uncategorized.yml

```diff
@@ -438,6 +422,7 @@
     - !type:CharacterJobRequirement
```

Comment:
For future reference, generic requirements like this could be replaced with a CharacterDepartmentRequirement to avoid repetition. Not requesting any changes, just a note for the future.

Reply:
In this case, it is specifically excluding the CMO and the medical intern, and in some cases also the chemist. You'll notice that these multi-job requirements were already here, I didn't add them.